ci-operator/config/openshift-kni/lifecycle-agent/openshift-kni-lifecycle-agent-main__nightly-4.15.yaml

```diff
@@ -3,6 +3,10 @@ base_images:
     name: "4.15"
     namespace: ocp-kni
     tag: lifecycle-agent-operator
+  tests:
+    name: "4.16"
```

Comment:
Is this the to version or from?

Reply:
neither, this is the version of the `openshift-tests` binary